man/systemd-cryptenroll.xml

```diff
@@ -409,6 +409,16 @@
         <xi:include href="version-info.xml" xpointer="v248"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tpm2-handle=</option><replaceable>HANDLE</replaceable></term>
+
+        <listitem><para>Configures which parent key to use for sealing, using the TPM handle of the key. The
+        default is the Storage Root Key (SRK) handle index <literal>0x81000001</literal>. This should not be
+        changed unless you know what you are doing.</para>
```

Comment:
should document expected range and syntax

Reply:
yep, updated the wording, thanks